src/lib.rs

```diff
@@ -200,11 +200,19 @@ impl From<SchedulingPolicy> for thread_priority::ThreadPriority {
     fn from(policy: SchedulingPolicy) -> Self {
         match policy {
             SchedulingPolicy::Other(_) | SchedulingPolicy::Idle | SchedulingPolicy::Batch(_) => {
-                thread_priority::ThreadPriority::Specific(0)
+                thread_priority::ThreadPriority::Crossplatform(
```

Comment:
the newer version of `ThreadPriority` removed `Specific` and replaced it with two variants. This is the only one that can be created. The only way to make it is with a `try_into` but we know we already in the correct range (0..=99) so this is correct.